package/setup.py

```diff
@@ -325,33 +370,24 @@ def extensions(config):
                                    'MDAnalysis/lib/formats/src/trr_seek.c',
                                    'MDAnalysis/lib/formats/src/xtc_seek.c',
                                    ],
-                          include_dirs=include_dirs + ['MDAnalysis/lib/formats/include',
+                          include_dirs=['MDAnalysis/lib/formats/include',
                                                        'MDAnalysis/lib/formats'],
                           define_macros=largefile_macros)
-    util = MDAExtension('lib.formats.cython_util',
+    util = Extension('lib.formats.cython_util',
                         sources=['MDAnalysis/lib/formats/cython_util' + source_suffix],
-                        include_dirs=include_dirs)
+                        include_dirs=[])
 
-    pre_exts = [dcd, dcd_time, distances, distances_omp, qcprot,
+    extensions = [dcd, dcd_time, distances, distances_omp, qcprot,
                   transformation, libmdaxdr, util]
-    cython_generated = []
-    if use_cython:
-        extensions = cythonize(pre_exts)
-        for pre_ext, post_ext in zip(pre_exts, extensions):
-            for source in post_ext.sources:
-                if source not in pre_ext.sources:
-                    cython_generated.append(source)
-    else:
-        #Let's check early for missing .c files
-        extensions = pre_exts
-        for ext in extensions:
-            for source in ext.sources:
-                if not (os.path.isfile(source) and
-                        os.access(source, os.R_OK)):
-                    raise IOError("Source file '{}' not found. This might be "
-                                "caused by a missing Cython install, or a "
-                                "failed/disabled Cython build.".format(source))
-    return extensions, cython_generated
+
+    #Let's check early for missing source files
```

Comment:
That check was actually added in #692 (where we introduced the automatic cython rebuild), just slightly modified here. My idea was that devs that relied on the cythonized files being distributed might start getting failed builds, and this was a neater way to give some hint as to why.